src/Writer.php

```diff
@@ -270,7 +270,8 @@ protected function renderEditor(Frame $frame): WriterContract
     {
         $file = $this->getFileRelativePath((string) $frame->getFile());
 
-        $this->render('at <fg=green>'.$file.'</>'.':<fg=green>'.$frame->getLine().'</>');
+        $line = $frame->getLine() ?? '0';
```

Comment:
maybe just cast the line? `(int) $frame->getLine()`

Reply:
That also works! Modified :)
Also added a comment to explain why.